examples/react-native/RunAnywhereAI/src/screens/ChatScreen.tsx

```
@@ -134,7 +134,68 @@ const registerChatTools = () => {
+          required: true,
+        },
+      ],
+    },
+    async (args) => {
+      const expression = (args.expression || args.input) as string;
+      console.log('[Tool] calculate called for:', expression);
+      try {
+        // Simple and safe math evaluation
+        const sanitized = expression.replace(/[^0-9+\-*/().% ]/g, '');
+        // eslint-disable-next-line no-eval
+        const result = eval(sanitized);
```

Biome (2.3.13)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)